plugins/single_plugins/scale.cpp

```diff
@@ -804,29 +814,30 @@ class wayfire_scale : public wf::plugin_interface_t
                     translation_x = x - vg.x + ((width - vg.width) / 2.0);
                     translation_y = y - vg.y + ((height - vg.height) / 2.0);
 
-                    scale_data[child].animation.scale_animation.scale_x.set(
-                        scale_data[child].transformer->scale_x,
-                        active ? child_scale_x : 1);
-                    scale_data[child].animation.scale_animation.scale_y.set(
-                        scale_data[child].transformer->scale_y,
-                        active ? child_scale_y : 1);
-                    scale_data[child].animation.scale_animation.translation_x.set(
-                        scale_data[child].transformer->translation_x,
+                    auto& view_data = scale_data[child];
+
+                    scale_x = scale_y = std::min(scale_x, scale_y);
```

Comment:
It looks like this line was added by mistake?

Reply:
Yes, indeed, thanks for noticing!